microceph/ceph/configwriter.go

```diff
@@ -87,7 +87,7 @@ auth allow insecure global id reclaim = false
 
 [client.radosgw.gateway]
 rgw init timeout = 1200
-rgw frontends = beast port={{.rgwPort}}
+rgw frontends = beast port={{.rgwPort}} {{.sslConfig}}
```

Comment:
This would expand linearly right ? can we rather do something like:
```
{{if .sslPort}}ssl_port = {{.sslPort}}{{end}}
{{if .sslCertificate}}ssl_certificate = {{.sslCertificate}}{{end}}
```

I believe this would be more readable.

Reply:
Thanks for the feedback, @UtkarshBhatthere! I adjusted the code based on that suggestion. There are two things to consider:
* I needed to put all of the variables in the same lines as the frontend line to make SSL work.
* I needed to restrict the `sslPort` variable to be rendered only if the certificate and private key were informed. Otherwise, as there is a default value for the SSL port and SSL needs the certificate and the private key, the frontend will fail to start.